.github/workflows/shop.yml

```diff
@@ -0,0 +1,85 @@
+name: CI for ecommerce / shop demo
+
+on:
+  push:
+    branches: [ "main" ]
+    paths:
+      - 'bank/**'
+      - '.github/workflows/shop.yml'
+  pull_request:
+    types:
+      - ready_for_review
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - 'bank/**'
```

Comment:
Shouldn't this be `e-commerce/**'?  Or am I misunderstanding it?

Reply:
Yes,